print_queue.py

```diff
@@ -0,0 +1,36 @@
+import time
+
+from constants import VERSION
+
+from print_job import PrintJob
+
+
+class PrintQueue(object):
```

Comment:
This used to be necessary, but it's not anymore in python 3.
```
class PrintQueue(object):
```
is equivalent to
```
class PrintQueue:
```

Reply:
Damn, I've merged already. I'll remove the unnecessary part in future work.